CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Change history for stripes-inventory-components
 
+## 1.0.1 IN PROGRESS
+
+- [UISINVCOMP-34](https://folio-org.atlassian.net/browse/UISINVCOMP-34) Get all locations in useCommonData hook when user is in consortium mode.
```

Comment:
`1.0.0` hasn't been released so we can include this fix there. No need to add `1.0.1` for now